islandora-interest-groups/Islandora-DevOps-Interest-Group
# documentation updates

## What does this Pull Request do?

Remove duplicate Ubuntu documentation under root since the updated Version exists under "Deployment Guides".

Quick follow up to last large PR.

Fix typo with backticks on lines 338 and 340.

Update Fits version and provide new link since old doesn't work anymore. 

Add --recursive to git clone of basic-solr-config since islandora_transforms is now a sub module. 

## What's new?

Fits version is now the same as islandora_vagrant. 

Other stuff is just fixing existing documentation.

Old copy of Ubuntu guide is not under root of this repo. It exists under Deployment Guides. 

## How should this be tested?

Try test installing Fits using provided steps.

Make sure fedora.sh file is populated with the appropriate variables after backticks are removed. 

Make sure that basic-solr-config clones properly with islandora_transforms. 

## Additional Notes

Interested parties

@DonRichards
